ui/src/warehouse.vue

```diff
@@ -52,9 +52,19 @@ export default {
                     sessionStorage.setItem('auth_redirect', document.location.href);
                     document.location = Vue.config.auth_signin;
                 } else {
-                    const sessionId = document.location.hash.substring(8);
+                    const urlFragment = document.location.hash;
+                    // we expect a hash of type: #ezbids=abc123&pipeline=DWI for example
+                    // pipeline may be undefined
+                    const [ ezbidsSession, pipeline ] = urlFragment.split("&");
```

Comment:
Instead of assuming position, I'd recommend using:
```js
document.location.hash = "#ezbids=abc123&pipeline=DWI";
const [ ezbids: ezbidsSession, pipeline ] = Object.fromEntries(
    document.location.hash
        .substring(1).split('&').map(s => s.split('='))
);
```

Reply:
done